tests/fast_reboot_dump_test.py

```diff
@@ -0,0 +1,49 @@
+import json
+import os
+from deepdiff import DeepDiff
+from utilities_common.db import Db
+import importlib
+fast_reboot_dump = importlib.import_module("scripts.fast-reboot-dump")
+
+class TestFastRebootDump(object):
+
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
+
+        test_db_dumps_directory = os.getcwd() + '/tests/fast_reboot_dump_dbs'
+        asic_db_object = Db()
+        app_db_object = Db()
+        asic_db = asic_db_object.db
+        app_db = app_db_object.db
+        populate_db(asic_db, test_db_dumps_directory, 'ASIC_DB.json')
+        populate_db(app_db, test_db_dumps_directory, 'APPL_DB.json')
+
+        cls.asic_db = asic_db
+        cls.app_db = app_db
+
+    def test_generate_fdb_entries_vlan_portcahnnel_member(self):
+        vlan_ifaces = ['Vlan2']
+
+        fdb_entries, all_available_macs, map_mac_ip_per_vlan = fast_reboot_dump.generate_fdb_entries_logic(self.asic_db, self.app_db, vlan_ifaces)
+
+        expectd_fdb_entries = [{'FDB_TABLE:Vlan2:52-54-00-5D-FC-B7': {'type': 'dynamic', 'port': 'PortChannel0001'}, 'OP': 'SET'}]
+        assert not DeepDiff(fdb_entries, expectd_fdb_entries, ignore_order=True)
+
+        expectd_all_available_macs = {('Vlan2', '52:54:00:5d:fc:b7')}
+        assert not DeepDiff(all_available_macs, expectd_all_available_macs, ignore_order=True)
+
+        expectd_map_mac_ip_per_vlan = {'Vlan2': {'52:54:00:5d:fc:b7': 'PortChannel0001'}}
+        assert not DeepDiff(map_mac_ip_per_vlan, expectd_map_mac_ip_per_vlan, ignore_order=True)
+    
```

Comment:
as the motivation is to verify that the dump script is working with this type of port channels but also with others should we add also additional port channel types to verify they are all used properly?

Reply:
a different type of port channel will be a routed interface, on this case we don't have fdb entries.